test/message/async_error_eval_cjs.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+require('../common');
+const fixtures = require('../common/fixtures');
+const errorModule = fixtures.path('async-error.js');
+const { spawnSync } = require('child_process');
+
+const main = `'use strict';
+
+const four = require('${errorModule}');
+
+async function main() {
+  try {
+    await four();
+  } catch (e) {
+    console.log(e);
+  }
+}
+
+main();
+`;
+
+// --eval CJS
+{
+  const child = spawnSync(process.execPath, [
+    '-e',
+    main
+  ], {
+    env: { ...process.env }
```

Comment:
Is this redundant ? The env default is set to `env: process.env` IIRC.

Reply:
It's easier to turn on/off other environment variables this way (e.g. NODE_DEBUG)